contrib/pai_vscode/README.md

```diff
@@ -48,7 +49,7 @@ After added a cluster configuration, you can find the cluster in *PAI CLUSTER EX
 Submit V1 job:
```

Comment:
there should be a blank line or a ```-``` if you want line break

Reply:
Done.